modules/extensions/src/views/lite/components/debugger/components/Predictions.tsx

```diff
@@ -23,22 +24,26 @@ const Predictions = (props: Props) => {
       <div className={style.sectionTitle}>Top Predictions</div>
       {Object.keys(predictions).map((key, index) => {
         const { confidence, intents } = predictions[key]
-        return (
-          <div className={style.subSection} key={index}>
-            <p>
-              {key} {formatConfidence(confidence)}%
-            </p>
-            <ul>
-              {intents.slice(0, 4).map(i => {
-                return (
-                  <li key={i.label}>
-                    <Intent name={i.label} confidence={i.confidence} />
-                  </li>
-                )
-              })}
-            </ul>
-          </div>
-        )
+        if ((key === 'global' && activePrompt) || key !== 'global') {
+          return (
+            <div className={style.subSection} key={index}>
```

Comment:
don't use index as key 

from the [react docs](https://reactjs.org/docs/lists-and-keys.html#keys) : 
```
We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an in-depth explanation on the negative impacts of using an index as a key. If you choose not to assign an explicit key to list items then React will default to using indexes as keys.
```

Reply:
changed index for topicName